docs/usage/setup_circle.md

```diff
@@ -0,0 +1,71 @@
+CircleCI is the tool we use for Continuous Integration, Continuous Delivery and Continuous Deployment.
+
+CircleCI automates the testing and deployment of your application. It runs whenever you change your application source code. You can configure configure how CircleCI works with the `circle.yml` file.
+
+If you choose to use CircleCI for your project you will need to host your source code on GitHub or Bitbucket.
+
+## Configure circle.yml
+
+You will need to add a circle.yml file to your applications root directory.
+
+Here is the basic structure of a circle.yml file.
+
+```
+---
+machine:
+  ruby:
+    version: 2.3.1
+
+dependencies:
+  override:
+    - bin/ciprepare.sh
+
+test:
+  override:
+    - bin/citest.sh
+
+deployment:
+  all:
+    branch: /.*/
+    commands:
+      - bin/cideploy.sh
+```
+
+The `circle.yml` file has four phases:
+
+- `machine:` specify what language the virtual machine needs to run your application. You can specify more than one.
+- `dependencies:` install any dependencies for your application
+- `test:` run your application tests
+- `deployment:` deploy your code to cloud.gov.au
+
+**CircleCI will not deploy your application if any of the above phases fail.**
+
+Read the [Circle documentation on configuring your circle.yml file](https://circleci.com/docs/configuration/)
+
+## Write CircleCI command scripts
+
+We recommend you write commands for each CircleCI phase in separate shell scripts. This is helpful if your team chooses to use another CI/CD tool in the future. If you prefer you can write commands in the `circle.yml` file.
+
+You can see an [example of how we setup scripts](https://github.com/AusDTO/jalpha/tree/master/template/bin) for a simple Jekyll project.
```

Comment:
Jalpha doesn't appear to have all the relevant scripts at the moment. Might need to update Jalpha with the new version of the scripts